odxtools/parameters/physicalconstantparameter.py

```diff
@@ -57,7 +80,7 @@ def decode_from_pdu(self, decode_state: DecodeState):
         return phys_val, next_byte_position
 
     def __repr__(self) -> str:
-        repr_str = f"PhysicalConstantParameter(short_name='{self.short_name}', physical_constant_value={self.physical_constant_value}"
+        repr_str = f"PhysicalConstantParameter(short_name='{self.short_name}', physical_constant_value={self.physical_constant_value!r}"
```

Comment:
What's the r for at the end?

Reply:
without it, `mypy` complains that the result of the f-string changed for `bytes` objects a few python versions ago (3.7?)